lib/pure/collections/tables.nim

```diff
@@ -2342,6 +2342,16 @@ proc inc*[A](t: var CountTable[A], key: A, val = 1) =
     if val != 0:
       insertImpl()
 
+proc dec*[A](t: var CountTable[A], key: A, val = 1) =
+  ## Decrements `t[key]` by `val` (default: 1).
+  runnableExamples:
+    var a = toCountTable("aaabbbbbbbbbbb")
+    a.dec('a')
+    a.dec('b', 10)
+    doAssert a == toCountTable("aab")
+
+  inc(t, key, -val)
```

Comment:
Alternative way is to extract the content of the  `inc` proc into a private templates. Then make both of the ` inc` and `dec` procs call that template. Otherwise, the `dec` proc can probably have some overhead compared to the `inc` proc.